src/libxrpl/protocol/ErrorCodes.cpp

```diff
@@ -116,7 +116,9 @@ constexpr static ErrorInfo unorderedErrorInfos[]{
     {rpcUNKNOWN_COMMAND,        "unknownCmd",           "Unknown method.", 405},
     {rpcORACLE_MALFORMED,       "oracleMalformed",      "Oracle request is malformed.", 400},
     {rpcBAD_CREDENTIALS,        "badCredentials",       "Credentials do not exist, are not accepted, or have expired.", 400},
-    {rpcTX_SIGNED,              "transactionSigned",    "Transaction should not be signed.", 400}};
+    {rpcTX_SIGNED,              "transactionSigned",    "Transaction should not be signed.", 400},
+    {rpcENTRY_NOT_FOUND,        "entryNotFound",        "Entry not found.", 400},
+    {rpcUNEXPECTED_LEDGER_TYPE, "unexpectedLedgerType", "Unexpected ledger type.", 400}};
```

Comment:
Could we add a trailing comma here and move `}
;` to the next line, so the code is easier to change the next time?
In some places C++ allows it, in some - not

Reply:
Fixed in [f380124](https://github.com/XRPLF/rippled/pull/5237/commits/f380124966119c33193e8b91fdb857d521f70953)